scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py

```diff
@@ -365,8 +374,42 @@ def config_lvm(self):
     # Check if cvg is non empty
     if not cvg:
         raise MotrError(errno.EINVAL, "cvg is empty\n")
+    return cvg_cnt, cvg
 
+def update_bgsize(self):
     dev_count = 0
+    lvm_min_size = None
+    lvm_min_size = None
+
+    cvg_cnt, cvg = get_cvg_cnt_and_cvg(self)
+    for i in range(int(cvg_cnt)):
+        cvg_item = cvg[i]
+        try:
+            metadata_devices = cvg_item["metadata_devices"]
+        except:
+            raise MotrError(errno.EINVAL, "metadata devices not found\n")
+        check_type(metadata_devices, list, "metadata_devices")
+        sys.stdout.write(f"\nlvm metadata_devices: {metadata_devices}\n\n")
+        for device in metadata_devices:
+            cmd = f"pvs --noheadings {device}"
+            vgname = (execute_command(self, cmd)[0]).split(sep=None)[1]
+            cmd = "lvdisplay | grep \"LV Path\" | grep {} | grep -v swap".format(vgname)
+            lv_list = (execute_command(self, cmd)[0]).replace("LV Path", '').split('\n')[0:-1]
+            for i in range(len(lv_list)):
+                lv_list[i] = "".join(lv_list[i].split())
+                lv_path = lv_list[i]
```

Comment:
One single line of code is enough?
```
lv_path = "".join(lv_list[i].split())
```

It's not a good idea to modify the list while we iterating it. And no need to do so.

Reply:
Done 683d315